mmengine/utils/misc.py

```diff
@@ -527,16 +527,17 @@ def get_object_from_string(obj_name: str):
             module_name = f'{module_name}.{part}'
         except StopIteration:
             # if obj is a module
-            return module
+            return obj
```

Comment:
If returning `obj` here, the case `get_object_from_string('torch')` will throw an error.